standalone/supervisor-conf/jmwalletd.conf

```diff
@@ -0,0 +1,10 @@
+[program:jmwalletd]
+directory=/src/scripts
+command=exec-wrapper.sh python jmwalletd.py
+autostart=false
+stdout_logfile=/root/.joinmarket/logs/jmwalletd_stdout.log
+stdout_logfile_maxbytes=5MB
+stdout_logfile_backups=0
```

Comment:
Just out of curiosity: why 0? Anything to be said against log rotation? Not that it matters much, just curious why it's set this way everywhere.

Reply:
No reason other than it is the same value as in btcpayserver's images for joinmarket.

We can change it to any sensible value you have in mind. I think anything between 0 - 5 is acceptable. What do you think?

Reply:
> One general question: We'll want to switch all the `regtest` settings to `mainnet` when we ship this, correct?

I want to emphasize again that no regtest settings are present during build time and in the result image. Everything can be controlled via environment variables (or overlay files - provide your own default config)! It will start with the default configuration if nothing is provided (hence, mainnet is default!).